### PR TITLE
getting started -> quickstart

### PR DESCRIPTION
**Pull request by balopat**
- small rewordings
- getting started guide will be quickstart now 
- quickstart focus on the flow with minikube - so I removed the default-repo references for now - we will have to explain that in another tutorial - or a proper "get started with your project" guide

### CODE REVIEW

File: docs/content/en/docs/quickstart/_index.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+[getting-started] Hello world!
+
+```
+
+`skaffold dev` monitors your code repository and perform a Skaffold workflow
+every time a change is detected. `skaffold.yaml` provides specifications of the
+workflow, which, in this example, is
+
+* Building a Docker image from the source using the Dockerfile
+* Tagging the Docker image with the Sha256 Hash of its contents
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
* Tagging the Docker image with the `sha256` hash of its contents
```

File: docs/content/en/docs/quickstart/_index.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+## Before you begin
+
+* [Install Skaffold]({{< relref "/docs/install" >}})
+* [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [Install minikube](https://minikube.sigs.k8s.io/docs/start/)
+
+{{< alert title="Note">}}
+Skaffold will build the app using the Docker daemon hosted inside minikube. 
+If you want to deploy against a different Kubernetes cluster, e.g. Kind, GKE clusters, you will have to install Docker (for Desktop) to build this app.
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
If you want to deploy against a different Kubernetes cluster, e.g. Kind, GKE clusters, you will have to install Docker to build this app.
```

File: docs/content/en/docs/quickstart/_index.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+ - pod/getting-started created
+Deploy complete in 374.060415ms
+Watching for changes...
+[getting-started] Hello world!
+[getting-started] Hello world!
+[getting-started] Hello world!
+
+```
+
+`skaffold dev` monitors your code repository and perform a Skaffold workflow
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
`skaffold dev` watches your local source code and executes your Skaffold pipeline
```

File: docs/content/en/docs/quickstart/_index.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+every time a change is detected. `skaffold.yaml` provides specifications of the
+workflow, which, in this example, is
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
workflow - in this example, the pipeline is
```

File: docs/content/en/docs/quickstart/_index.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+func main() {
+	for {
+		fmt.Println("Hello Skaffold!")
+		time.Sleep(time.Second * 1)
+	}
+}
+```
+
+The moment you save the file, Skaffold will repeat the workflow described in
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
When you save the file, Skaffold will see this change and repeat the workflow described in
```

File: docs/content/en/docs/quickstart/_index.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+`skaffold.yaml` and eventually re-deploy your application. Once the pipeline
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
`skaffold.yaml`, rebuilding and redeploying your application. Once the pipeline
```

File: docs/content/en/docs/quickstart/_index.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+is completed, you should see updated outputs in the terminal:
```
Comment (nkubala) [inline docs/content/en/docs/quickstart/_index.md]:
```suggestion
is completed, you should see the changes reflected in the output in the terminal:
```

**Comment by balopat** [comment]
@nkubala can you take another look?

**Pull request merged**